homework/hw03-2.py

```diff
@@ -0,0 +1,18 @@
+
+from mrjob.job import MRJob
+import re
+
+class ReviewCount(MRJob):
+
+    def mapper(self, _, line):
+        col = line.split(',')
```

Comment:
Should've used csv.reader() first to get the exact answer. The correct answer is 85 (-1)

Reply:
though csv.reader() is better, this answer is technically correct, so i'll give this point back